traits/stubs_tests/util.py

```diff
@@ -108,7 +108,7 @@ def run_mypy(filepath):
         dest_filename = os.path.basename(filepath)
         dest = shutil.copyfile(filepath, os.path.join(tempdir, dest_filename))
         normal_report, error_report, exit_status = mypy_api.run(
-            [dest, '--show-error-code'])
+            [dest, '--show-error-code', '--follow-imports=skip'])
```

Comment:
The extra argument here was necessary to prevent errors from files _other_ than the one under test. Eventually we should be able to fix the main Traits codebase up enough that we no longer need this, but it may take some time.